ref/src/interpolate.f90

```diff
@@ -0,0 +1,160 @@
+module interpolate
+  implicit none
+
+contains
+
+  subroutine interpolate_allocate(odims, interpFactor, fdims)
```

Comment:
Consider renaming this routine because it doesn't actually allocate anything.  It simply computes the dimensions of the interpolated array.

Reply:
Done.